[fetcheus] go-velocity does not work

```
(send *ri* :go-velocity 0.1 0 0.5)
[ERROR] [1571647540.324682228]: Setting velocity in y-axis is not valid for non-holonomic robot
nil
```

cc @Affonso-Gui  @YutoUchimi

--- a/jsk_fetch_robot/fetcheus/fetch-interface.l
+++ b/jsk_fetch_robot/fetcheus/fetch-interface.l
@@ -209,7 +209,7 @@ Example: (send self :gripper :position) => 0.00"
     (x y d ;; [m/sec] [m/sec] [rad/sec]
        &optional (msec 1000) ;; msec is total animation time [msec]
        &key (stop t) (wait t))
-    (unless (not (zerop y))
+    (unless (zerop y)
       (ros::ros-error
         "Setting velocity in y-axis is not valid for non-holonomic robot")
       (return-from :go-velocity nil))

--- a/jsk_fetch_robot/fetcheus/test/test-fetcheus.l
+++ b/jsk_fetch_robot/fetcheus/test/test-fetcheus.l
@@ -116,6 +116,16 @@
     (assert (v< (send robot :head :angle-vector) #f(30 30)))
     ))
 
+(deftest fetch-go-velocity
+  (let (go-vel-arg)
+    (setq go-vel-arg (list 0.1 0 0.5))
+    (assert (send* *ri* :go-velocity go-vel-arg)
+            (format nil ":go-velocity doesn't return t with argument ~A" go-vel-arg))
+    (setq go-vel-arg (list 0.1 0.1 0.5))
+    (assert (not (send* *ri* :go-velocity go-vel-arg))
+            (format nil ":go-velocity returns t with argument ~A" go-vel-arg))
+    ))
+
 
 (run-all-tests)
 (exit)

